recipes/StitchM/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "StitchM" %}
+{% set version = "1.0.1" %}
+{% set import_name = "stitch_m" %}
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DiamondLightSource/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 34363f186e12df1fe035b8f01a352eb979d9148ed7ac698a34da3726990dabd0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - {{ name }} = {{ import_name }}.scripts.commandline:main
+  skip: True  # [py<35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  host:
+    - python
+    - pip
+    - numpy >=1.17.4
```

Comment:
Are you linking against NumPy? If so, please add a C compiler and follow the directions https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
I had an import that was breaking a script being created without numpy in host. I've fixed it now and moved numpy to run requirements.